pkg/lib/v0_2_0/keptn.go

```diff
@@ -83,3 +87,106 @@ func (k *Keptn) GetShipyard() (*Shipyard, error) {
 	}
 	return &shipyard, nil
 }
+
+// SendCloudEvent sends a cloudevent to the event broker
+func (k *Keptn) SendCloudEvent(event cloudevents.Event) error {
+	if k.UseLocalFileSystem {
+		log.Println(fmt.Printf("%v", string(event.Data())))
+		return nil
+	}
+
+	return k.EventSender.SendEvent(event)
+}
```

Comment:
Isn't this a duplicate of https://github.com/keptn/go-utils/blob/67ac0f2ba06dd11f3e045f7ff7053674175fbbce/pkg/lib/v0_2_0/events.go#L123-L131

Reply:
And it's not used in this PR either as far as I can see.

Reply:
this function was moved because I figured it would make sense to have the SendCloudEvent method in the same place where the `SendStarted/StatusChanged/Finished` methods are